libs/solid/slideshow/src/lib/components/slideshow-select/slideshow-select.component.html

```diff
@@ -5,9 +5,17 @@
     [routerLink]="[slideshow.id]"
   >
     <img
+      *ngIf="slideshow.title_image !== null; else NoImg"
       [src]="slideshow.title_image.img.thumbnail"
       [alt]="slideshow.title_image.img_alt"
     />
+    <ng-template #NoImg>
+      <img
+        src="assets/icons/icon.svg"
+        alt="No image available"
+        style="width: 6rem; height: 6rem"
```

Comment:
Styling should not be coded into the html file. Each component has a scss-file - that is where all css/scss belongs.
Please move this part to `slideshow-select.component.scss`